spec/ruby/optional/capi/module_spec.rb

```diff
@@ -339,4 +339,15 @@ def ruby_test_method
       @m.rb_class2name(CApiModuleSpecs::M).should == "CApiModuleSpecs::M"
     end
   end
+
+  describe "rb_mod_ancestors" do
+    it "returns an array of ancestors" do
+      module One
+      end
+      module Two
+        include One
+      end
```

Comment:
This will define modules `Object::One` and `Object::Two` (at the toplevel).
It's better to use `Module.new` to avoid that problem.

Reply:
Fixed, thanks.